code/modules/martial_arts/martial.dm

```diff
@@ -4,8 +4,8 @@
 /datum/martial_art
 	var/name = "Martial Art"
 	var/streak = ""
-	var/max_streak_length = 6
 	var/temporary = FALSE
+	var/owner_UID
```

Comment:
This could prob be a hard reference at one point since a martial_arts should always have an owner if used. But this would require better management of the martial_arts by the mind datum first. Prob not worth doing in this PR